src/pretalx/orga/templates/orga/submission/feedbacks_list.html

```diff
@@ -0,0 +1,27 @@
+{% extends "orga/cfp/base.html" %}
+{% load i18n %}
+{% load rich_text %}
+
+{% block content %}
+    <h2 class="d-flex w-100 justify-content-between align-items-start">
+        {% trans "Event feedback" %}
+    </h2>
+
+    {% regroup feedback|dictsort:"talk.title" by talk.title as feedbacks_list %}
+
+    {% for submission in feedbacks_list %}
+        <h4>{% trans "Feedback for submission" %} <a href="{{ submission.list.1.talk.orga_urls.base }}">{{ submission.grouper }}</a></h4>
```

Comment:
Let's drop the "submission" here, and just go with "Feedback for", and then use `{{ quotation_open }}` and `{{ quotation_close }}` surrounding the title.